libtransmission/peer-mgr.cc

```diff
@@ -514,6 +514,7 @@ class tr_swarm
                 tr_announcerAddBytes(tor, TR_ANN_UP, event.length);
                 tor->set_date_active(tr_time());
                 tor->session->add_uploaded(event.length);
+                msgs->bytes_sent_to_peer.add(tr_time(), event.length);
```

Comment:
*nit*: This line should be in `tr_peerMsgsImpl::did_write()`, since it only concerns the peer.